.github/workflows/integration-tests-on-release.yaml

```diff
@@ -64,10 +64,10 @@ jobs:
         name: ceph-helm-suite-artifact
         path: /home/runner/work/rook/rook/tests/integration/_output/tests/
 
-    - name: setup tmate session for debugging
-      if: failure()
-      uses: mxschmitt/action-tmate@v3
-      timeout-minutes: 30
+    # - name: setup tmate session for debugging
```

Comment:
why not just remove it? we never log in the session during release right?

Reply:
right, removed